podium-gateway/src/main/webapp/app/shared/request/request-review-feedback.ts

```diff
@@ -12,8 +12,12 @@ import { RequestReviewDecision } from './request-review-decision';
 
 export class RequestReviewFeedback {
 
-    decision?: RequestReviewDecision;
-    feedback?: string;
+    id?: string;
+    advice?: RequestReviewDecision;
+    reviewer?: string;
```

Comment:
This will be mapped in a next PR as a user representation so we'll have the fields we need at hand